elements/bulbs-video/bulbs-video.js

```diff
@@ -57,22 +58,26 @@ export default class BulbsVideo extends BulbsElement {
   }
 */
   render () {
+    let allProps = {
+      ...this.props,
+      ...this.state,
+      actions: this.store.actions,
+      autoplayNext: typeof this.props.twitterHandle === 'string',
+      disableLazyLoading: typeof this.props.disableLazyLoading === 'string',
+      disableMetaLink: typeof this.props.disableMetaLink === 'string',
+      embedded: typeof this.props.embedded === 'string',
+      enablePosterMeta: typeof this.props.enablePosterMeta === 'string',
+      muted: typeof this.props.muted === 'string',
+      noEndcard: typeof this.props.noEndcard === 'string',
+    };
+
+    if (!allProps.disableLazyLoading) {
+
+      return loadOnDemand(BulbsVideoRoot)(allProps);
```

Comment:
This is the main change. If the `bulbs-video` instance has not been provided a `disable-lazy-loading` property, the video element will be subject to lazy loading.